i18n/zh-Hans/docusaurus-plugin-content-docs/current/guides/cases/connect-mqtt.md

```diff
@@ -22,7 +22,24 @@ spec:
   protocol: MQTT
   protocolSettings:
     MQTTSetting:
-      MQTTTopic: /test/test # 修改此值
+```
+
+在 `examples/mqttDeviceShifu/mqtt_deploy/mqtt_deviceshifu_configmap.yaml` 中:
+
+```yml
+...
+data:
+  driverProperties: |
+    driverSku: testMQTT
+    driverImage: 
+  instructions: | # 一条命令可以订阅一个topic，如需订阅多个topic，增加命令即可
+    instructions:
+      get_topicmsg1: # 修改此值
+        protocolPropertyList:
+          MQTTTopic: "/test/test1" # 修改此值
+      get_topicmsg2: # 订阅多个topic时修改此值
```

Comment:
The comment seems confusing, each instruction corresponds to one topic, we cannot subscribe multiple topics with one instruction